docs/cluster-deployment/deploy-to-cluster-azure/signup-azure.rst

```diff
@@ -69,6 +73,7 @@ wizard and, finally, to deploy your cluster. This will be explained in the
 :ref:`next section of the tutorial <configure-azure-to-cluster>`.
 
 
+.. _documentation on subscription plans: https://crate.io/docs/cloud/reference/en/latest/subscription-plans.html
 .. |this link| raw:: html
```

Comment:
unrelated to this change: but didn't we come up with a way to avoid doing this?

Reply:
Did we? If so I do not recall it

Reply:
I believe that site-wide CSS was added to do this for all external links

Reply:
Ah you mean the HTML! Sorry, I was looking at the line above it. Yeah those are legacy things that I'll clean up soon, but updating the docs is higher priority